Jenkinsfile_nightly

```diff
@@ -0,0 +1,13 @@
+#!groovy
+@Library("Infrastructure")
+
+def type = "java"
+def product = "ccd"
+def component = "spring-boot-template"
+
+withNightlyPipeline(type, product, component) {
+  enableFortifyScan('ccd-aat')
```

Comment:
no need parameter, as product is already ccd.

Reply:
product is actually aac.